.travis.yml

```diff
@@ -1,18 +1,21 @@
 language: python
 sudo: false
 python:
-  - 2.6
   - 2.7_with_system_site_packages
+  - 3.4
```

Comment:
Remind me: are we explicitly unsupporting Python 3.3, or is it just inconvenient to include it in the Travis versions?